src/components/simulation/simulation-view.tsx

```diff
@@ -87,13 +87,13 @@ export const SimulationView: React.FC<IProps> = (props) => {
       <TransitionGroup>
         { fishAnimations.map((animation, index) =>
             <CSSTransition
-              key={index}
+              key={`fish-transition-${environment}-${index}`}
               timeout={isRunning ? kSimulationOneWeekPeriodInMilliseconds : 0}
               classNames="animation-item"
             >
               <SimulationAnimation
                 animation={animation}
-                key={`fish-animation-${index}`}
+                key={`fish-animation-${environment}-${index}`}
```

Comment:
In the end I don't think these key changes were necessary, but I decided to leave them in.